### PR TITLE
Update content versions page

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Update content versions page in accordance with our new content buckets system

### CODE REVIEW

File: src/content/contributing/translation-program/content-versions/index.md [hunk 1]
```diff
@@ -41,34 +42,95 @@ Language v2.0 translates high-impact, highly trafficked pages on ethereum.org an
...
+## Use case pages {#use-case-pages}
+
+- [Decentralized autonomous organizations (DAOs)](/dao/)
+- [Decentralized finance (DeFi)](/defi/)
+- [Non-fungible tokens (NFT)](/nft/)
+- [Translation Program](/contributing/translation-program/)
+- [Translation guide](/contributing/translation-program/translation-guide/)
+- [Ethereum grants](/community/grants/)
+
+## Foundation developer docs {#foundational-docs}
```
Comment (wackerow) [inline src/content/contributing/translation-program/content-versions/index.md:53]:
```suggestion
## Foundational developer docs {#foundational-docs}
```

File: src/content/contributing/translation-program/content-versions/index.md [hunk 2]
```diff
@@ -41,34 +42,95 @@ Language v2.0 translates high-impact, highly trafficked pages on ethereum.org an
...
+
+- [Introduction README](/developers/docs/)
```
Comment (wackerow) [inline src/content/contributing/translation-program/content-versions/index.md:55]: We recently changed the name of this to Overview
```suggestion
- [Overview](/developers/docs/)
```

File: src/content/contributing/translation-program/content-versions/index.md [hunk 3]
```diff
@@ -41,34 +42,95 @@ Language v2.0 translates high-impact, highly trafficked pages on ethereum.org an
 - [Language support](/languages/)
 - [Brand assets](/assets/)
```
Comment (minimalsm) [inline src/content/contributing/translation-program/content-versions/index.md:43]:
```suggestion
```

File: src/content/contributing/translation-program/content-versions/index.md [hunk 4]
```diff
@@ -41,34 +42,95 @@ Language v2.0 translates high-impact, highly trafficked pages on ethereum.org an
...
+- [Validium](/developers/docs/scaling/validium/)
 
+## Additional Learn and general Pages {#general-pages}
+
+- [About ethereum.org](/about/)
 - [Ethereum Improvement Proposals (EIPs)](/eips/)
 - [Enterprise Ethereum](/enterprise/)
 - [Private Ethereum](/enterprise/private-ethereum/)
 - [History](/history/)
 - [Glossary](/glossary/)
```
Comment (minimalsm) [inline src/content/contributing/translation-program/content-versions/index.md:132]:
```suggestion
- [Glossary](/glossary/)
- [Brand assets](/assets/)
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Love it... nice job @minimalsm!
Just notes a couple small things

File: src/content/contributing/translation-program/content-versions/index.md [hunk 5]
```diff
@@ -39,36 +40,97 @@ Language v2.0 translates high-impact, highly trafficked pages on ethereum.org an
...
+- [Miner extractable value (MEV)](/developers/docs/mev/)
+- [Orcles](/developers/docs/oracles/)
+- [Scaling](/developers/docs/scaling/)
+- [Layer 2 Rollups](/developers/docs/scaling/layer-2-rollups/)
+- [State channels](/developers/docs/scaling/state-channels)
+- [Sidechains](/developers/docs/scaling/sidechains)
+- [Plasma](/developers/docs/scaling/plasma)
+- [Validium](/developers/docs/scaling/validium/)
 
+## Additional Learn and general Pages {#general-pages}
```
Comment (samajammin) [inline src/content/contributing/translation-program/content-versions/index.md:124]:
```suggestion
## Additional Learn and general pages {#general-pages}
```

**Review by samajammin — APPROVED** [review]
🏆

File: src/content/contributing/translation-program/content-versions/index.md [hunk 6]
```diff
@@ -39,36 +40,97 @@ Language v2.0 translates high-impact, highly trafficked pages on ethereum.org an
 - [Developer learning tools](/developers/learning-tools/)
```
Comment (minimalsm) [inline src/content/contributing/translation-program/content-versions/index.md:40]:
```suggestion
- [Developer's home](/developers/)
- [Developer learning tools](/developers/learning-tools/)
```

**Review by wackerow — APPROVED** [review]

**Pull request merged**